diff --git a/package.json b/package.json
index 6e98e65..08853c9 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "mas",
   "version": "1.0.0",
-  "homepage": ".",
+  "homepage": "https://rgarrido03.github.io/mas",
   "private": true,
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^6.2.1",
diff --git a/src/index.jsx b/src/index.jsx
index 2ff3a8c..0733cba 100644
--- a/src/index.jsx
+++ b/src/index.jsx
@@ -54,7 +54,7 @@ root.render(
         <meta name="theme-color" content="#0066CC" />
       </Helmet>
     </HelmetProvider>
-    <Router> {/* Add basename='/mas' for GitHub Pages deployment */}
+    <Router basename='/mas'> {/* Add basename='/mas' for GitHub Pages deployment */}
       <Routes>
         <Route path="/" element={<ActorChoosing />} />
         <Route path="/client/home" element={<ClientHome />} />
